switch_model/transmission/transport/build.py

```diff
@@ -348,7 +355,7 @@ def post_solve(instance, outdir):
             "BuildTx": value(mod.BuildTx[tx, p]) if (tx, p) in mod.BuildTx else ".",
```

Comment:
Shall we return 0 instead of "." here? That would mean that there was not Tx added.

Reply:
I think the . gives more information. "." means building is not allowed. 0 means it's allowed but nothing was built.

Reply:
mmm. If it is now allowed to build, maybe we can remove it from this file? We specify which Tx are allowed to build and which not. For this file might be more beneficial to only include ones that actually are build?

Reply:
The catch is this file contains other information such as efficiency etc. which has a value for each period, even those when building is not allowed. So we want to keep all the periods. If you're concerned about graphing, in the latest version of the graphing tools you can do `tools.get_dataframe('transmission.csv', convert_dot_to_na=True)`. I added this as it's pretty common in switch for dots to be na.